tests/imap_tests.in

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export EMAILS=""
+export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
+export FACTS_SOURCE="$TEST_DIR/img_test.lf"
+export IMAGE_RARITY=1
+export IMG_ASSET_DIR="${RUN_DIR}/tests/img_test_img"
+export LF_IMAGE_MAP="${RUN_DIR}/assets/lf_imap.json"
```

Comment:
This should probably point to a test-specific version of the file.

Reply:
Will do when it's actually a test specific version